cluster/addons/iso-client/cputopology/cpu.go

```diff
@@ -0,0 +1,82 @@
+package cputopology
+
+import "sync"
+
+// CPU struct contains information about logical CPU.
+type CPU struct {
+	// SocketID represents socket where CPU is.
+	SocketID int
+	// CoreID represents core where CPU is.
+	CoreID int
+	// CPUID is unique CPU number.
+	CPUID int
+	// IsIsolated stores information is CPU is not use by OS scheduler.
+	IsIsolated bool
+	// IsInUse stores information
+	IsInUse bool
```

Comment:
An alternative to storing this here could be to maintain a collection of Binding (pod => []CPU) in the isolator.